go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.10.0
-	github.com/vishvananda/netlink v1.3.0
+	github.com/vishvananda/netlink v1.3.1-0.20250303224720-0e7078ed04c8
```

Comment:
v1.3.1 hasn't been released yet. This uses the latest commit at the time of creating this PR: https://github.com/vishvananda/netlink/commit/0e7078ed04c84cea47daea45be061544e565ec49

Reply:
We could handle the error in the code here: https://github.com/vishvananda/netlink/commit/0e7078ed04c84cea47daea45be061544e565ec49

with something like:
```go
		if errors.Is(err, unix.EINTR) || err != nil {
			return fmt.Errorf("could not list addresses: %v", err)
		}
```
but we might end up ignoring unix.EINTR's which really matter.